docker-compose.yml

```diff
@@ -19,15 +19,26 @@ version: "3.8"
 
 
 services:
+  management:
+    build:
+      context: .
+      dockerfile: Dockerfile-management
```

Comment:
What does this container really do? It doesn't spin any process.

Reply:
move a major management logic there, so it is also unavailable when tiles serve (similar to `tileserver`)

so in update.sh you can find major usage of it